operations/tasks/main.yml

```diff
@@ -1,3 +1,6 @@
 ---
 - include: upgrade-docker.yml
   when: osbs_upgrade_docker
+
+- include: update-node_selector-labels.yml
+  when: osbs_node_selector_labels
```

Comment:
Let's follow the pattern of using a boolean, `osbs_label_node`, to control whether or not the sub-operation tasks run.
This will allow us to remove all managed labels from a node without adding a new one.
We should use an empty list as the default for `osbs_node_selector_labels`.

Reply:
ok